support/support.d.ts

```diff
@@ -0,0 +1,10 @@
+import {IOptions} from "../src/Interfaces/IOptions";
+import {Container} from "../src/Classes/Container";
+
+declare class ParticlesJS {
+    static load(tagId: string, params: IOptions): Container;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class method 'load' must be marked either 'private', 'public', or 'protected'](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5052388)